src/store/modules/accounts/index.js

```diff
@@ -57,7 +57,7 @@ export default {
     },
 
     signTransaction({ getters: { active, getModule }, dispatch }, txBase64) {
-      return dispatch(`${getModule(active).name}/signTransaction`, txBase64);
+      return dispatch(`${getModule(active).name}/signTransaction`, txBase64, {});
```

Comment:
Why is that needed?

Reply:
```
async signTransaction({ dispatch }, {
      txBase64,
      opt: { modal = true, host = null },
    }
  ```
  If not supplied then the default assignments of the modal and host throw error.